<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDatabase</class>
 <widget class="QDialog" name="dlgDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration file database</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="editFind">
     <property name="placeholderText">
      <string>Find</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblCount">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="btnFind">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="qtocc.qrc">
       <normaloff>:/icon/find.png</normaloff>:/icon/find.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="btnRefreshAll">
     <property name="toolTip">
      <string>Save all profiles in the table to the latest version</string>
     </property>
     <property name="text">
      <string>Refresh all</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTableWidget" name="tableDatabase">
     <property name="columnCount">
      <number>1</number>
     </property>
     <column/>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QTableWidget" name="tableDatabaseFind">
     <property name="columnCount">
      <number>1</number>
     </property>
     <column/>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tip: Simply place the configuration file in the Database directory in the root of the app.&lt;/p&gt;&lt;p&gt;Double-click the entry to open it and generate a new EFI folder on the desktop.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qtocc.qrc"/>
 </resources>
 <connections/>
</ui>
